core/src/com/fallenflame/game/LevelSelectMode.java

```diff
@@ -13,17 +13,14 @@
 
 public class LevelSelectMode implements Screen, InputProcessor {
 
-    /** Textures temporarily loaded in so I have something to look at as I set up this class */
-    private static final String PLAY_BTN_FILE = "textures/fireplay.png";
-    private Texture playButton = new Texture(PLAY_BTN_FILE);
-    private static final String BACKGROUND_FILE = "textures/firelevelselect.png";
+    private static final String BACKGROUND_FILE = "textures/ls_background.png";
     private Texture background = new Texture(BACKGROUND_FILE);
 
-    private static final String LEVEL_BTN_FILE = "textures/particle.png";
+    private static final String LEVEL_BTN_FILE = "textures/ls_unlocked_level.png";
     private Texture levelButton = new Texture(LEVEL_BTN_FILE);
 
     /** Position vectors for all the level select buttons */
-    private Vector2[] posVec = {new Vector2(50,200),new Vector2(150,300),new Vector2(250,225),new Vector2(350,100),new Vector2(425,250),new Vector2(500,375),new Vector2(550,150),new Vector2(625,300),new Vector2(700,100),new Vector2(750,250)};
+    private Vector2[] posVec = {new Vector2(200,400),new Vector2(300,400),new Vector2(400,400),new Vector2(500,400),new Vector2(600,400),new Vector2(200,200),new Vector2(300,200),new Vector2(400,200),new Vector2(500,200),new Vector2(600,200)};
```

Comment:
What might be worthwhile is, instead of doing this, make it increments of canvas.getWidth() so if we change the width for any reason, this doesn't completely fall apart. . So maybe it's (canvas.getWidth()/10)*x, where x is its position in the row (or, if you want the array, index % 5).

Reply:
Agreed. Was just thinking we should make the screen larger